samples/apps/copilot-chat-app/WebApp/src/components/chat/chat-list/ChatList.tsx

```diff
@@ -51,10 +53,7 @@ export const ChatList: FC = () => {
                         Conversations
                     </Text>
                     <div>
-                        {/* TODO: Allow users to filter conversations by name
-                        <Button
-                        icon={<Filter20Regular />}
-                        appearance="transparent" /> */}
+                        <Button icon={<Filter20Regular />} appearance="transparent" />
```

Comment:
This (and line 47) was removed in a previous PR since we don't have filter support yet, make sure your branch is updated with main

Reply:
Will rebase